packages/replay/test/integration/errorSampleRate.test.ts

```diff
@@ -369,6 +369,52 @@ describe('Integration | errorSampleRate', () => {
       ]),
     });
   });
+
+  it('stops replay when session expires', async () => {
```

Comment:
We only want this for the error mode, right? Do we have tests in the session recording mode that check the expected behavior there?

Reply:
I think this existing test should cover this: https://github.com/getsentry/sentry-javascript/blob/master/packages/replay/test/integration/session.test.ts#L106